qa/tasks/cephfs/test_multifs.py

```diff
@@ -0,0 +1,255 @@
+                mount_x.client_remote, keyring))
+
+        return keyring_paths
+
+    def setup_fs_contents(self, cephfs_mntpt, filenames, filedata):
+        filepaths = []
+        iter_on = zip((self.mount_a, self.mount_b), filenames, filedata)
+
+        for mount_x, filename, data in iter_on:
+            if cephfs_mntpt != '/' :
+                mount_x.run_shell(args=['mkdir', cephfs_mntpt])
+                filepaths.append(os_path_join(mount_x.hostfs_mntpt,
+                                              cephfs_mntpt, filename))
+            else:
+                filepaths.append(os_path_join(mount_x.hostfs_mntpt, filename))
+
+            mount_x.write_file(filepaths[-1], data)
+
+    def remount_with_new_client(self, cephfs_mntpt, filenames,
+                                           keyring_paths):
+        if isinstance(cephfs_mntpt, str) and cephfs_mntpt != '/' :
+            cephfs_mntpt = '/' + cephfs_mntpt
+
+        self.mount_a.remount(client_id=self.client_id,
+                             client_keyring_path=keyring_paths[0],
+                             client_remote=self.mount_a.client_remote,
+                             cephfs_name=self.fs1.name,
+                             cephfs_mntpt=cephfs_mntpt,
+                             hostfs_mntpt=self.mount_a.hostfs_mntpt,
+                             wait=True)
+        self.mount_b.remount(client_id=self.client_id,
+                             client_keyring_path=keyring_paths[1],
+                             client_remote=self.mount_b.client_remote,
+                             cephfs_name=self.fs2.name,
+                             cephfs_mntpt=cephfs_mntpt,
+                             hostfs_mntpt=self.mount_b.hostfs_mntpt,
+                             wait=True)
+
+        return (os_path_join(self.mount_a.hostfs_mntpt, filenames[0]),
+                os_path_join(self.mount_b.hostfs_mntpt, filenames[1]))
+
+
+class TestClientsWithoutAuth(TestMultiFS):
+
+    def test_mount(self):
+        # setup part...
+        keyring = self.fs2.authorize(self.client_id, ('/', 'rw'))
+        keyring_path = self.create_keyring_file(self.mount_a.client_remote,
+                                                keyring)
+
+        # functionality to be tested...
+        retval = self.mount_a.remount(client_keyring_path=keyring_path,
+                                      check_status=False)
+
+        # tests...
+        self.assertIsInstance(retval, tuple)
+        self.assertEqual(len(retval), 3)
+        self.assertIsInstance(retval[0], CommandFailedError)
+        if 'kernel' in str(type(self.mount_a)).lower():
+            # TODO: when mon caps for the Ceph FS are not assigned to client,
+            # the mount.ceph prints "no mds server or cluster laggy" instead
+            # of permission denied. (But it prints "permission denied", when
+            # file is absent MON caps are assigned and MDS caps are absent).
```

Comment:
Don't follow this part of the sentence, "when file is absent MON caps are assigned and MDS caps are absent"

Reply:
Verified what I meant originally, added a new test and fixed the TODO. Thanks!